src/configuration/services/mysql.md

```diff
@@ -79,6 +79,71 @@ foreach ($relationships['database'] as $endpoint) {
 2. MySQL Errors such as "PDO Exception 'MySQL server has gone away'" are usually simply the result of exhausting your existing diskspace. Be sure you have sufficient space allocated to the service in [.platform/services.yaml](/configuration/services.md).
 
 
+## Multiple databases
+
+On the 10.0 version and later of MariaDB it is possible to define multiple databases as well as multiple users with different permissions.  To do so requires defining multiple endpoints.  Under the `configuration` key of your service there are two additional keys:
```

Comment:
You're right. What do you think about referring explicitly to the "MariaDB service"? To me the current wording implies that this is a feature that was added to the MariaDB software itself. If you dont see that as a native speaker, fine by me.